chart/templates/statsd/statsd-deployment.yaml

```diff
@@ -104,8 +104,17 @@ spec:
             - "--statsd.mapping-config=/etc/statsd-exporter/mappings.yml"
           {{- end }}
           resources: {{- toYaml .Values.statsd.resources | nindent 12 }}
-          {{- with .Values.statsd.env }}
-          env: {{- toYaml . | nindent 12 }}
+          env:
```

Comment:
If no environment you will have

```python
          env:

```
without any value. I this is is valid chart but don't think its good.

Reply:
There's empty list of value in default value metrics.
So maybe it'll be empty list if user did not intensively set env without any value.

Please let me know if this is wrong or bad case

https://github.com/apache/airflow/blob/4d65f216dec5705f82262d9c4a9f799e92b55944/chart/values.yaml#L1924

Reply:
My bad, looking like it's a good practice to do so 😄